style.css

```diff
@@ -133,49 +139,641 @@ header {
 .social-media-list {
   list-style: none;
   display: flex;
-  width: 170.17px;
-  height: 20px;
+  width: 200px;
+  height: 40px;
   flex: none;
   order: 3;
   flex-grow: 0;
 }
 
-#twitter-icon {
+ul li {
+  list-style-type: none;
+}
+
+.social-media-list > li {
+  margin-right: 10px;
+}
+
+.twitter-icon {
   position: absolute;
   left: 6.4%;
   right: 88.27%;
   top: 77.81%;
   bottom: 18.88%;
 }
 
-#linkedin-icon {
+.linkedin-icon {
   position: absolute;
   width: 20px;
   height: 20px;
   left: 63px;
   top: 470px;
 }
 
-#m-icon {
+.m-icon {
   position: absolute;
   left: 27.47%;
   right: 67.2%;
   top: 78.15%;
   bottom: 19.21%;
 }
 
-#github-icon {
+.github-icon {
   position: absolute;
   left: 38.13%;
   right: 57.07%;
   top: 77.81%;
   bottom: 18.87%;
 }
 
-#free-icon {
+.free-icon {
   position: absolute;
   left: 48%;
   right: 48.22%;
   top: 77.81%;
   bottom: 18.87%;
 }
+
+.works-section {
+  display: flex;
```

Comment:
- [ ] The main container of the projects has a `display:grid`, please kindly fix that 😉😉